README.md

```diff
@@ -5,7 +5,7 @@
     <a href="https://pypi.org/project/linkstatus"><img alt="Python Versions" src="https://img.shields.io/pypi/pyversions/linkstatus.svg?style=flat"></a>
     <a href="https://travis-ci.com/digitronik/linkstatus"><img alt="Build Status"
```

Comment:
@raukadah please move this to `pythonpune` as i removed from my repo
https://travis-ci.com/pythonpune/linkstatus
https://api.travis-ci.com/pythonpune/linkstatus.svg?branch=master